internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/fastclient/meta/RequestBasedMetadataZstdIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package com.linkedin.venice.fastclient.meta;
+
+import static org.testng.Assert.*;
+
+import com.linkedin.venice.compression.CompressionStrategy;
+import com.linkedin.venice.compression.VeniceCompressor;
+import com.linkedin.venice.fastclient.stats.ClusterStats;
+import com.linkedin.venice.integration.utils.VeniceRouterWrapper;
+import com.linkedin.venice.meta.ReadOnlyStoreRepository;
+import com.linkedin.venice.utils.TestUtils;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.testng.annotations.Test;
+
+
+public class RequestBasedMetadataZstdIntegrationTest extends RequestBasedMetadataIntegrationTest {
```

Comment:
Can we merge this with `RequestBasedMetadataIntegrationTest`? Having two test classes means we need to set up the cluster two times where we could have done with just one